app/controllers/DeclarationController.scala

```diff
@@ -59,7 +59,11 @@ class DeclarationController @Inject()(
 
   def onPageLoad(waypoints: Waypoints): Action[AnyContent] = (identify andThen getData() andThen requireData) {
     implicit request =>
-      isReportDuplicate(request.pstr, 2023)
+      val currentTaxYear = request.userAnswers.get(TaxYearPage) match {
+        case Some(value) => value
+        case _ => TaxYear("2022")
```

Comment:
which tax year needed here? 2022 or 2023

Reply:
It doesn't really matter. It's just for me to get some info in QA from the logging.